user-profile/src/components/CandidatePersonalInfo/CandidatePersonalInfo.js

```diff
@@ -80,7 +84,7 @@ function CandidatePersonalInfo() {
         setPhone(val);
     }, []);
 
-    const url = 'http://192.168.0.128:8080/api/personal-information'
+    const url = 'http://35.168.113.87:8080/api/personal-information'
```

Comment:
We should replace the base url with an environment variable e.g https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env